### PR TITLE
Issue1763

### PR DESCRIPTION
**Pull request by tonypark0403**
Add white space indicators in the editor. #1763

### CODE REVIEW

**Comment by Pomax** [comment]
@tonypark0403 a few things: this PR looks like it needs a rebase before it can be merged, but I'm also missing the linkback to the specific issue this addresses (the github issue that your push comment refers to is itself a closed PR, so that doesn't seem quite right). Also, can you update the title so that people looking at the `/pulls` list can immediately see what it's for?

File: src/config.json [hunk 1]
```diff
@@ -20,57 +20,76 @@
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
```
Comment (humphd) [inline src/config.json]: `git checkout master src/config.json` please

File: src/extensions/default/brackets-show-whitespace/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# EditorConfig.org
```
Comment (humphd) [inline src/extensions/default/brackets-show-whitespace/.editorconfig]: Delete this file

File: src/extensions/default/brackets-show-whitespace/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Changelog
```
Comment (humphd) [inline src/extensions/default/brackets-show-whitespace/CHANGELOG.md]: Delete this file

File: src/extensions/default/brackets-show-whitespace/main.js [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+            _applyColors();
+        }
+    }
+
+    
+    // --- Loaders and Unloaders ---
+
+    function loadPreferences() {
+        _preferences = PreferencesManager.getExtensionPrefs(preferencesID);
+        _preferences.definePreference("enabled", "boolean", defaultPreferences.enabled);
```
Comment (humphd) [inline src/extensions/default/brackets-show-whitespace/main.js]: I think we should ponder this--do we want this on by default?  cc @flukeout

File: src/extensions/default/brackets-show-whitespace/styles/main.less [hunk 5]
```diff
@@ -0,0 +1,76 @@
+/*
```
Comment (humphd) [inline src/extensions/default/brackets-show-whitespace/styles/main.less]: Let's convert this to a `.css` file and modify the code above that loads it.  We dont' want to render LESS files in the browser.

File: src/extensions/default/brackets-show-whitespace/styles/whitespace-colors-css.tmpl [hunk 6]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (humphd) [inline src/extensions/default/brackets-show-whitespace/styles/whitespace-colors-css.tmpl]: What is a `.css.tmpl` file?

**Comment by tonypark0403** [comment]
I changed the branch name to be more clear so please delete this branch.

**Comment by Pomax** [comment]
There's only one person who can delete branches on your repository, and that's you: go to your own repository on github, and then click the "branches" tab. This lets you delete branches from your repository.

**Pull request closed without merging**